src/TextareaAutosize.js

```diff
@@ -98,10 +98,14 @@ export default class TextareaAutosize extends React.Component {
 
 }
 
-let onNextFrame = window.requestAnimationFrame;
+if (process.browser) {
```

Comment:
Does it work with webpack? I think more reliable would be to move check for `requestAnimationFrame` within the `onNextFrame` function.